src/plugins/map/MQ2MapCommands.cpp

```diff
@@ -24,6 +24,14 @@
 
 #include <fmt/format.h>
 
+const int maxOptionsSize = 100;
+std::stack<MapFilterOption*> optionStack;
```

Comment:
I don't like this as a global, it is only used in AddMapFilterOptionAsImGuiSetting, and its state outside of that function should always be empty. pass it in as a parameter instead.